docs/blog/2020-04-27-building-Serverless-site-faster-with-Gatsby/index.md

```diff
@@ -14,6 +14,7 @@ Since 2015 Serverless has been providing a platform to unify all the tools softw
 Any new front-end technologies had to mesh well with the existing site’s architecture. “The Serverless site has a blog and a documentation section, both of which pull Markdown files from GitHub repositories,” says Narayan. “We needed a way to pull those Markdown files while still maintaining a very fluent experience for the content contributors.”
 
 ## Ready to (re)launch with Gatsby 🚀
+
```

Comment:
Prettier was throwing errors for the missing the line breaks that are added here. @blainekasten Let me know if you'd like me to remove them.